feegrant/index.md

```diff
@@ -0,0 +1,254 @@
+```
+
+Note that Bob has only `kudos` tokens and won't be able to pay for any fees that might be needed.
+
+Add Carol and an initial `stake` balance to the genesis file:
+
+```bash
+simd add-genesis-account carol 1000000stake --keyring-backend test
+```
+
+Generate a transaction to add Alice to the initial validator set:
+
+```bash
+simd gentx alice 1000000stake --chain-id demo
+```
+
+Add the validator transaction to the genesis file:
+
+```bash
+simd collect-gentxs
+```
+
+## Start Chain
+
+You are now ready to start a single node network on your local machine.
+
+Start the chain:
+
+```bash
+simd start
+```
+
+## Grant Allowance
+
+Before Bob can send `kudos` to Carol, you must set up an allowance for Bob so that Carol pays for any gas fees the transaction might incur. Because Alice is the validator, it does not make sense to make her the granter because a validator receives all the fees that the granter paid for. If Alice were the granter, there would not be a difference in her balance after a transaction was completed.
+
+The `BasicAllowance` is a permission for a grantee to use up fees until the `spend_limit` or `expiration` is reached. Create an allowance with a spend limit of `100000stake` and no expiration date:
+
+```bash
+simd tx feegrant grant $CAROL_KEY $BOB_KEY --from carol --spend-limit 100000stake
+```
+
+View the allowance:
+
+```bash
+simd q feegrant grants $BOB_KEY
+```
+
+## Send Tokens
+
+First, let's check the balances of Alice, Bob, and Carol. Verifying the initial balance provides a baseline so that you can later confirm if your transaction was successful:
+
+```bash
+simd q bank balances $ALICE_KEY
+simd q bank balances $BOB_KEY
+simd q bank balances $CAROL_KEY
+```
+
+Note that Alice has `4999000000stake` because she bonded `1000000stake` to become a validator during the chain setup.
+
+Any transaction that is sent using the `tx` command can use the `--fee-account` flag to specify an account as input to pay for the fees.
+
+Send `kudos` tokens from Bob to Carol, while Carol pays the fees:
+
+```bash
+simd tx bank send $BOB_KEY $CAROL_KEY 100kudos --from bob --fee-account $CAROL_KEY --fees 500stake
+```
+
+Look at the balances again:
+
+```bash
+simd q bank balances $ALICE_KEY
+simd q bank balances $BOB_KEY
+simd q bank balances $CAROL_KEY
+```
+
+Notice how Carol no longer has the `500stake` that was added to the transaction that Bob signed.
+
+View the allowance again:
+
+```bash
+simd q feegrant grants $BOB_KEY
+```
+
+Note how `spend_limit` has been reduced and Bob now has 99500stake left to spend on fees.
+
+## Revoke Allowance
+
+The granter can revoke the allowance from the grantee using the `revoke` command.
+
+Revoke allowance:
+
+```bash
+simd tx feegrant revoke $CAROL_KEY $BOB_KEY --from carol
+```
+
+View the allowance:
+
+```bash
+simd q feegrant grants $BOB_KEY
+```
+
+## 🎉 Congratulations 🎉
+
+By completing this tutorial, you have learned how to use the feegrant module:
```

Comment:
Another one for consistency.

```suggestion
By completing this tutorial, you have learned how to use the fee grant module:
```

Reply:
Thanks Ryan! I guess this is another question then: are we sticking with capitals when referring to the module?

```suggestion
By completing this tutorial, you have learned how to use the Fee Grant module:
```

Reply:
I would have written as "feegrant" so not entirely sure. I think consistency is the most important thing so if capitalized here, just make sure you do so elsewhere. Maybe @barriebyron could weigh in?